### PR TITLE
fix: refactor HTTP error codes for PUT /mentorship_relation/{request_id}/task/{task_id}/comment/{comment_id}

### LINKED ISSUE #634: Fix: refactor HTTP error codes PUT /mentorship_relation/{request_id}/task/{task_id}/comment/{comment_id}
## Description
As a GSoC student,
I need the correct HTTP error response code to be returned accordingly by Mentorship System API in relation to PUT /mentorship_relation/{request_id}/task/{task_id}/comment/{comment_id},
so that I can represent the error message accordingly to both Mentorship System and Bridge In Tech users.

## Mocks
NA

### Update [Required]
- [ ] 403: Forbidden needs to be returned instead of 400: Bad Request if it has unaccepted state relation or the relation is not created by the user.

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by RiddhiAthreya**
### Description

Made changes to the HTTP error codes being returned according to what's discussed in the issue. I have also made the appropriate changes in the swagger documentation.

Fixes  #634

### Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Screenshot (172)

Screenshot (173)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] Update Swagger documentation 

**Code/Quality Assurance Only**

- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by vj-codes** [comment]
@RiddhiAthreya the linked issue is a `first timers only` issue, and you have already made your first contribution through #1126. Please check the issue labels while claiming next time :)

**Review by vj-codes — CHANGES_REQUESTED** [review]
@RiddhiAthreya please resolve the merge conflicts

File: .flake8 [hunk 1]
```diff
@@ -1,13 +0,0 @@
-[flake8]
-ignore =
-    # W503: line break before binary operator
-    # ignored as it creates inconsistency with black
-    W503,
-    # E501: line too long (> 88 characters)
```
Comment (devkapilbansal) [inline .flake8:6]: Why this file is deleted?

This is important for flake8 to work correctly.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -7,16 +7,16 @@
 * When developing a new feature, include at least one test when applicable.
 * When submitting a PR, please follow [this template](PULL_REQUEST_TEMPLATE.md) (which will probably be already filled up once you create the PR).
 * When submitting a PR with changes to user interface (e.g.: new screen, or an email template), please add screenshots to the PR description.
-* When you are finished with your work, please squash your commits otherwise we will squash them on your PR (this can help us maintain a clear commit history).
+* When you are finished with your work, please squash your commits otherwise we will squash them on your PR (this can help us maintain a clear commit history).
```
Comment (devkapilbansal) [inline .github/CONTRIBUTING.md:10]: There should not be any space at the end. Trim the whitespace please

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -7,16 +7,16 @@
...
-* When you are finished with your work, please squash your commits otherwise we will squash them on your PR (this can help us maintain a clear commit history).
+* When you are finished with your work, please squash your commits otherwise we will squash them on your PR (this can help us maintain a clear commit history). 
 * When writing in-code documentation as Docstrings, please follow [this guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings).
 * When creating an issue to report a bug in the project, please follow our [bug_report.md](https://github.com/anitab-org/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) template.
 
 ## General Guidelines
 
 * If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](http://www.systers.io).
-* In an active repository (not an archived one), choose an open issue from the issue list, claim it in the comments, and a maintainer will assign it to you.
+* In an active repository (not an archived one), choose an open issue from the issue list, claim it in the comments, and a maintainer will assign it to you.
```
Comment (devkapilbansal) [inline .github/CONTRIBUTING.md:17]: Same as above. Same for similar cases as well

File: .isort.cfg [hunk 4]
```diff
@@ -1,9 +0,0 @@
-[tool.isort]
```
Comment (devkapilbansal) [inline .isort.cfg:1]: This should not be deleted too :sweat_smile:

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@RiddhiAthreya I see you remove the changes done for pre-commit. Please revert those changes.

**Comment by devkapilbansal** [comment]
If you are unsure how this will be done, then feel free to ask :smile:

**Comment by devkapilbansal** [comment]
@RiddhiAthreya are you working on this?

**Comment by isabelcosta** [comment]
I will close this PR and unassign due to lack of response.
Thank you for your contribution @RiddhiAthreya so far :)

**Pull request closed without merging**